### PR TITLE
job table ,job details, event table

### PR DESCRIPTION
**Pull request by Shimaa-Mahmoud-Abdelazeem**

### CODE REVIEW

File: src/pages/Events/event-details.html [hunk 1]
```diff
@@ -0,0 +1,477 @@
...
+              <i class="menu-icon mdi mdi-file-document"></i>
+              <span class="menu-title">Documentation</span>
+            </a>
+          </li>
+        </ul>
+      </nav>
+      <!-- partial -->
+      <div class="main-panel">
+        <div class="content-wrapper">
+            <h1 class="pt-5">Event Details</h1>
```
Comment (Nora-Lootfy) [inline src/pages/Events/event-details.html]: This is the first thing, i have noticed in your code.

Take it as an advice from me, don't ever make two h1 tag in your design, This is considered as bad practice. 
A little trick, if you want to display it as h1 use bootstrap class. 

Actually i like to display it a little smaller, The following code snippet looks much better
 ```html
<h2 class="pt-5  fw-bold">Event Details</h2>
```

File: src/pages/Events/events.html [hunk 2]
```diff
@@ -0,0 +1,804 @@
...
+            </a>
+          </li>
+        </ul>
+      </nav>
+      <!-- partial -->
+
+      <!--Task 13 Event Table created by Shimaa Mahmoud -->
+      <div class="main-panel">
+        <div class="content-wrapper ">
+          <h1 class="pt-5">Events</h1>
```
Comment (Nora-Lootfy) [inline src/pages/Events/events.html]: Same as event details

File: src/pages/Events/events.html [hunk 3]
```diff
@@ -0,0 +1,804 @@
...
+          <div class="row py-2">
```
Comment (Nora-Lootfy) [inline src/pages/Events/events.html]: Actually i prefer that search is on the same line as heading, use the code snippet
```html
<div class="row py-3 justify-content-between align-items-center">
              <h2 class="fw-bold col-lg-auto">Events</h2>
              <div class="col-lg-auto">
                <!-- Search Bar start -->
                <div class="search-bar">
                  <form action="">
                    <div
                      class="row g-1 justify-content-lg-end justify-content-start"
                    >
                      <div class="col-6 col-lg-3 form-floating">
                        <input type="text" class="form-control" id="title" />
                        <label for="title">Event title</label>
                      </div>
                      <div class="col-6 col-lg-3 form-floating">
                        <input type="date" class="form-control" id="date" />
                        <label for="date">Start date</label>
                      </div>
                      <div class="col-6 col-lg-3 form-floating">
                        <select class="form-control bg-white">
                          <option>-</option>
                          <option>categroy #1</option>
                          <option>categroy #2</option>
                          <option>categroy #3</option>
                          <option>categroy #4</option>
                        </select>
                        <label for="date">Select Category</label>
                      </div>
                      <button
                        class="col-auto btn border-0 btn-md"
                        type="submit"
                      >
                        <i class="icon-search fs-5"></i>
                      </button>
                    </div>
                  </form>
                </div>
                <!-- Search Bar ends -->
              </div>
            </div>
```

File: src/pages/Events/events.html [hunk 4]
```diff
@@ -0,0 +1,804 @@
...
+            
+              </div>
+              <!-- Search Bar ends -->
+            </div>
+            <div class="col-lg-12 grid-margin stretch-card">
+              <div class="card">
+                <div class="card-body">
+                  <h4 class="card-title">Events</h4>
+                  
+                  <div class="table-responsive content">
```
Comment (Nora-Lootfy) [inline src/pages/Events/events.html]: Actually, I like that you care a lot about responsive design, thanks for that

pretty cool, table😊

**Review by Nora-Lootfy — CHANGES_REQUESTED** [review]
Hello Shimaa,

Thanks for your hard working, very obvious for me that you have worked a lot.

I have left some comments for you, and add little tweaks that i forget to mention in the task so please replace them and make another pull request. 

Another thing about naming, I like to make all my files working with the same naming convention not just the code, so please lowercase the folder or files if you make any of them upper case.

I have also notice that you don't indent your code properly, but it is actually simple and make your code more neat, there is a plugin or extension called Prettier that formats your code just download and when you want to format your code press ctrl + alt + f

Last thing, I feel very comfortable when i see your pages, looks so pretty for me so thanks again.

I am pretty confident, we have amazing frontend developer 😊✨
Thanks ya Shimaa

[frontend.zip](https://github.com/eduCyber-HRmagazine/adminDashboard-frontend/files/14720058/frontend.zip)

**Comment by Shimaa-Mahmoud-Abdelazeem** [comment]
Thank you alot Nora for your comments, advice and encouraging words .
I will make all of changes that you are mentioned and push my code again.

**Review by Nora-Lootfy — APPROVED** [review]

**Pull request merged**